contracts/go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/onflow/flow-ft/contracts
 go 1.14
 
 require (
+	github.com/dapperlabs/flow-emulator v0.4.0 // indirect
```

Comment:
Is `github.com/dapperlabs/flow-emulator` needed as a dependency here? Unfortunately it's breaking CI because it' still a private repo. 😢 

Maybe try running `go mod tidy` in this repo?